march_hardware_builder/src/hardware_builder.cpp

```diff
@@ -246,9 +265,12 @@ std::unique_ptr<march::PowerDistributionBoard> HardwareBuilder::createPowerDistr
       boot_shutdown_byte_offsets["masterOk"].as<int>(), boot_shutdown_byte_offsets["shutdown"].as<int>(),
       boot_shutdown_byte_offsets["shutdownAllowed"].as<int>());
 
-  return std::make_unique<march::PowerDistributionBoard>(march::Slave(slave_index, pdo_interface, sdo_interface),
-                                                         net_monitor_offsets, net_driver_offsets,
-                                                         boot_shutdown_offsets);
+  std::shared_ptr<march::PowerDistributionBoard> pdb =
+      std::make_unique<march::PowerDistributionBoard>(march::Slave(slave_index, pdo_interface, sdo_interface),
```

Comment:
Why are you using `make_unique` if you want to create a `shared_ptr`?

Reply:
Oopsie :sweat_smile: